Add academic indicator: Thematic Persistence

PetrosStav opened the pull request:
@vtraag i created this new academic indicator based on what we do in the "EMERGING TOPICS Case Study" for topic analysis and topic persistence.

vtraag commented:
Hi @PetrosStav, thanks for this! It is my impression that there is much more literature that looks into emerging topics, and this approach is certainly not the only one. In order to provide a fair overview of such a type of indicator, it would be good to not only focus on the one that has been studied in one of our case studies, but also to provide the other relevant approaches / indicators.

PetrosStav commented:
Hi @vtraag, I revised this indicator and added some literature that looks into topic persistence, made the description + measurement more general and moved the formulas for what was used in the case studies into the methodology from SciNoBo.

vtraag commented:
Thanks, that looks a lot better! Could you follow standard reference practices as the other indicators? You can add the necessary BibTex to `references.bib`, and then cite them using `[@cite_this]`? See https://quarto.org/docs/authoring/citations.html.

vtraag commented:
Btw, it seems you've now also included here the reproducibility indicator. I presume this should be removed from this PR in favour of PR #93 , right?

vtraag commented:
I've merged the references, and cleaned it up a bit, I think it's good to go as is now. Thanks!
